.github/workflows/SinfoniaOperator.yml

```
@@ -18,16 +19,19 @@ jobs:
-        run: dotnet build --configuration Release
+        run: dotnet build ./ProjectCS/SymphonyKillChordProjectCS/SymphonyKillChordProjectCS/SymphonyKillChordProjectCS.csproj --configuration Release
 
       - name: 実行
         env:
-          NOTION_TOKEN: ${{ secrets.NOTION_TOKEN }}
-          NOTION_DATABASE_ID: ${{ secrets.NOTION_DATABASE_ID }}
           DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
-          NOTION_DATABASE_DATE_PROPERTY: ${{ secrets.NOTION_DATABASE_DATE_PROPERTY }}
-        run: dotnet run --project ./ProjectCS/SymphonyKillChordProjectCS/SymphonyKillChordProjectCS.csproj --configuration Release
+          NOTION_TOKEN: ${{ secrets.NOTION_TOKEN }}
+          NOTION_DATABASE_ID: ${{ NOTION_DATABASE_ID }}
```

actionlint (1.7.8)

undefined variable "NOTION_DATABASE_ID". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)